install/update_91_92.php

```diff
@@ -648,6 +648,180 @@ function update91to92() {
+                 `value` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
+                 PRIMARY KEY (`id`),
+                 KEY `olalevels_id` (`olalevels_id`)
+               ) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;";
+         $DB->queryOrDie($query, "9.2 add table glpi_olalevelactions");
+      }
+
+      if (!$DB->tableExists('glpi_olalevelcriterias')) {
+         $query = "CREATE TABLE `glpi_olalevelcriterias` (
+                 `id` int(11) NOT NULL AUTO_INCREMENT,
+                 `olalevels_id` int(11) NOT NULL DEFAULT '0',
+                 `criteria` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
+                 `condition` int(11) NOT NULL DEFAULT '0' COMMENT 'see define.php PATTERN_* and REGEX_* constant',
+                 `pattern` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
+                 PRIMARY KEY (`id`),
+                 KEY `olalevels_id` (`olalevels_id`),
+                 KEY `condition` (`condition`)
+               ) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;";
+         $DB->queryOrDie($query, "9.2 add table glpi_olalevelcriterias");
+      }
+
+      if (!$DB->tableExists('glpi_olalevels')) {
+         $query = "CREATE TABLE `glpi_olalevels` (
+                 `id` int(11) NOT NULL AUTO_INCREMENT,
+                 `name` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
+                 `olas_id` int(11) NOT NULL DEFAULT '0',
+                 `execution_time` int(11) NOT NULL,
+                 `is_active` tinyint(1) NOT NULL DEFAULT '1',
+                 `entities_id` int(11) NOT NULL DEFAULT '0',
+                 `is_recursive` tinyint(1) NOT NULL DEFAULT '0',
+                 `match` char(10) COLLATE utf8_unicode_ci DEFAULT NULL COMMENT 'see define.php *_MATCHING constant',
+                 `uuid` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
+                 PRIMARY KEY (`id`),
+                 KEY `name` (`name`),
+                 KEY `is_active` (`is_active`),
+                 KEY `olas_id` (`olas_id`)
+               ) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;";
+         $DB->queryOrDie($query, "9.2 add table glpi_olalevels");
+      }
+
+      if (!$DB->tableExists('glpi_olalevels_tickets')) {
+         $query = "CREATE TABLE `glpi_olalevels_tickets` (
+                    `id` int(11) NOT NULL AUTO_INCREMENT,
+                    `tickets_id` int(11) NOT NULL DEFAULT '0',
+                    `olalevels_id` int(11) NOT NULL DEFAULT '0',
+                    `date` datetime DEFAULT NULL,
+                    PRIMARY KEY (`id`),
+                    KEY `tickets_id` (`tickets_id`),
+                    KEY `olalevels_id` (`olalevels_id`),
+                    KEY `unicity` (`tickets_id`,`olalevels_id`)
+                  ) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;";
+         $DB->queryOrDie($query, "9.2 add table glpi_olalevels_tickets");
+      }
+
+      $query = "INSERT INTO `glpi_crontasks`
```

Comment:
What I told you was to use the `Crontask::register` method that will automatically check if the task already exists before adding it.
Currently; this method is working only for plugins... I've changed that in one of my recent PR; that has not yet been merged (will be today tought -- see https://github.com/glpi-project/glpi/pull/2438/files#diff-1454c423049813362d0918fb1f2dc346R1387)